bergzand/RIOT
# sys/usb: add basic CDC ACM support

### Contribution description

This PR adds a basic USB CDC ACM support.

Add the following modules to your application's Makefile:
```
USEMODULE += usbdev
USEMODULE += sam_usb
USEMODULE += usb_cdc
USEMODULE += plumbum
```

I tested this with SAML21-XPRO which require extra commits but this is out of the scope for now.
If you want to check the branch I'm playing with, see [1].

You can comment on this PR if you have any questions :)

### Testing procedure
Connect the dedicated ttyACMx with putty, check if loopback works.





[1] https://github.com/dylad/RIOT/tree/testdelete
